<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>950</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Soul Sync</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>50</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>soul_sync_logo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>250</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: #2C3E50;
}</string>
    </property>
    <property name="text">
     <string>Soul Sync</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="journal_textedit">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>420</y>
      <width>721</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    background-color: #F5F7FA; 
    color: #333333;
    font-size: 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    border: 2px solid #4682B4; 
    border-radius: 8px; 
    padding: 10px;
}


QTextEdit:focus {
    border: 2px solid #5A9BD6; 
    background-color: #FFFFFF; 
}</string>
    </property>
    <property name="placeholderText">
     <string>Take a deep breath... Whatâ€™s on your mind right now?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>850</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 14px;
    font-weight: bold;
    border: none;
    border-radius: 12px; 
    padding: 10px 20px;
    min-width: 100px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}


QPushButton:hover {
    background-color: #5A9BD6;
}


QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_btn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>850</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 14px;
    font-weight: bold;
    border: none;
    border-radius: 12px; 
    padding: 10px 20px;
    min-width: 100px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}


QPushButton:hover {
    background-color: #5A9BD6;
}


QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menu_btn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>850</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 14px;
    font-weight: bold;
    border: none;
    border-radius: 12px; 
    padding: 10px 20px;
    min-width: 100px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}


QPushButton:hover {
    background-color: #5A9BD6;
}


QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>375</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    font-size: 15px;
    font-weight: bold;
    color: #2C3E50; 
    padding: 10px;
    border-bottom: 2px solid #5A9BD6; 
    text-align: center;
}
</string>
    </property>
    <property name="text">
     <string>My Journal</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>312</y>
      <width>731</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="error_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>731</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
